src/node.cpp

```diff
@@ -27,9 +27,10 @@
 
 namespace ovms {
 
-Node::Node(const std::string& nodeName, uint32_t demultiplyCount) :
+Node::Node(const std::string& nodeName, uint32_t demultiplyCount, std::set<std::string> gatherFromNode) :
     nodeName(nodeName),
-    demultiplexCount(demultiplyCount ? std::optional<uint32_t>(demultiplyCount) : std::nullopt) {
+    demultiplexCount(demultiplyCount ? std::optional<uint32_t>(demultiplyCount) : std::nullopt),
+    gatherFrom(!gatherFromNode.empty() ? std::optional<std::set<std::string>>(gatherFromNode) : std::nullopt) {
```

Comment:
Do we need to use optional of set of strings? Maybe we could rely on a fact that empty set means no gathering at all.

Reply:
While this suggestion makes sens I think this is out of scope of this MR.